### PR TITLE
Filtering out alternate rows.

### PR DESCRIPTION
**Pull request by lakinwecker**
This allows us to more quickly filter out alternate rows based on
values that are included in column A for alternate rows.

### CODE REVIEW

**Comment by lakinwecker** [comment]
This Fixes #116

File: src/league.js [hunk 1]
```diff
@@ -158,7 +158,20 @@ league_attributes = {
...
                         return;
                     }
-                    if (_.isEqual(row['teams'].value.toLowerCase(), 'alternates')) {
+                    // TODO: this could be put into the config.
+                    var alternateRowSentinels = [
+                        'alternates',
+                        'alt is taken/unresponsive',
+                        'alt needs to reach 20 classical games'
+                    ];
+                    var teamName = row['teams'].value.toLowerCase();
```
Comment (endrawes0) [inline src/league.js]: This is misleading. Can we say potential team name or something?

**Comment by lakinwecker** [comment]
Name updated.

**Comment by endrawes0** [comment]
Bravo

**Pull request merged**